docs/modules/ROOT/pages/reference/arch-control-plane.adoc

```diff
@@ -0,0 +1,399 @@
+
+In the <<Organization Origin Configuration>> we have a label selector for the Compositions configured, which is used to select the Compositions available to the organization.
+This automatically includes which XRDs are available.
+
+Access to list the available XRDs and Compositions on cluster scope is denied to users of the CCP.
+See <<Listing>> on how the service catalog is made available.
+
+.Example Composition
+[source,yaml]
+----
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  labels:
+    metadata.appcat.vshn.io/serviceprovider: exoscale
+    metadata.appcat.vshn.io/zone: de-fra-1
+    metadata.appcat.vshn.io/servicename: VSHNPostgreSQL
+    metadata.appcat.vshn.io/description: PostgreSQL by VSHN
+    metadata.appcat.vshn.io/displayname: PostgreSQL by VSHN
+    metadata.appcat.vshn.io/end-user-docs-url: https://vs.hn/vshn-postgresql
+    metadata.appcat.vshn.io/product-description: https://products.docs.vshn.ch/products/appcat/postgresql.html
+  name: de-fra-1.exoscale.vshnpostgres.vshn.appcat.vshn.io
+spec:
+  compositeTypeRef:
+    apiVersion: vshn.appcat.vshn.io/v1
+    kind: XVSHNPostgreSQL
+  mode: Pipeline
+  pipeline:
+  - step: patch-and-transform
+    functionRef:
+      name: function-patch-and-transform
+    input:
+      apiVersion: pt.fn.crossplane.io/v1beta1
+      kind: Resources
+      resources:
+      - name: claim-transfer-to-ccsp
+        base:
+          apiVersion: kubernetes.crossplane.io/v1alpha2
+          kind: Object
+          spec:
+            forProvider:
+              manifest:
+                apiVersion: vshn.appcat.vshn.io/v1
+                kind: VSHNPostgreSQL
+            providerConfigRef:
+                name: exoscale-de-fra-1
+        patches:
+        - type: FromCompositeFieldPath
+          fromFieldPath: "TODO" <1>
+          toFieldPath: spec.forProvider.manifest
+  writeConnectionSecretsToNamespace: syn-crossplane
+----
+<1> We want to transfer the whole manifest - unclear as of now how to access it
+
+As we need several combinations of Compositions where only the CSPCP connection details are different they are generated via a Project Syn Commodore Component.
+
+=== Listing
+
+The available service catalog is subject to the organization, and specifically it's origin configuration.
+Therefore, the catalog is made available in the organization's context as a namespaced object called `catalog`.
+
+A CRD `Catalog` is provided. Instances of this object are reconciled into the organization namespaces.
+
+This allows to query the available services using `kubectl -n myorg get catalog`.
+
+The catalog reconciler reads the <<Organization Origin Configuration, OrganizationOriginConfig>> for catalog configuration, according to the Organization origin found on the label of the Namespace.
+
+[mermaid]
+....
+flowchart TD
+    A[Start] --> B[Get all namespaces with label<br/>appuio.io/resource.type=organization]
```

Comment:
Rendering of this looks broken -> the label details aren't visible in `make preview`

Reply:
Solved by rendering as PNG instead of SVG :man_shrugging: